components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -71,8 +71,12 @@ spec:
             drop:
               - all
         env:
+          - name: TYK_GW_DBAPPCONFOPTIONS_NODEISSEGMENTED
+            value : "false"
           - name: TYK_GW_LISTENPORT
             value: "{{ .Values.gateway.containerPort }}"
+          - name: TYK_GW_USEDBAPPCONFIGS
```

Comment:
If we set this environment variable here, component chart will get configured for Pro setup by default.

Reply:
Fixed

Reply:
will this field be true?